autogpt_platform/backend/backend/copilot/db.py

```
@@ -143,6 +144,9 @@ async def add_chat_message(
         "role": role,
         "sequence": sequence,
     }
+    if msg.get("id") is not None:
```

Ruff (0.15.9)

[error] Undefined name `msg`

(F821)